src/main/IO/Process/Services/Avatars/AvatarRepository.cs

```diff
@@ -157,12 +157,67 @@ private async Task CreateSqliteDatabasesAsync(string id)
                 var sqlStatements = File.ReadAllText(sqlFile);
                 var sqliteFileName = $"{Path.GetFileNameWithoutExtension(sqlFile)}.db";
 
-                using (var connection = new SqliteConnection($@"Data Source=file:{Path.Combine(id, sqliteFileName)}"))
-                {
-                    connection.Open();
+                using var connection = new SqliteConnection($@"Data Source=file:{Path.Combine(avatarItem.Id, sqliteFileName)}");
+
+                connection.Open();
 
-                    using var cmd = new SqliteCommand(sqlStatements, connection);
-                    await cmd.ExecuteNonQueryAsync();
+                if (sqliteFileName == "events.db")
+                {
+                    #region Notification Table Insertion
+
+                    string sqlInsertionCommand = @"
+BEGIN TRANSACTION;
+INSERT OR REPLACE INTO ""Notification"" (""SequenceId"", ""Timestamp"", ""TypeName"", ""Id"", ""Version"", ""AuthorId"", ""Data"")
```

Comment:
I changed "INSERT OR IGNORE" to "INSERT OR REPLACE" since the values are now dynamic.